### PR TITLE
Updates streamline collaboration usage triggers

### PR DESCRIPTION
**Pull request by nzaytsev**
# Description

# Checklist

- [ ] I have followed the guidelines in the Contributing document
- [ ] My changes follow the coding style of this project
- [ ] My changes build without any errors or warnings
- [ ] My changes have been formatted and linted
- [ ] My changes include any required corresponding changes to the documentation (including CHANGELOG.md and README.md)
- [ ] My changes have been rebased and squashed to the minimal number (typically 1) of relevant commits
- [ ] My changes have a descriptive commit message with a short title, including a `Fixes $XXX -` or `Closes #XXX -` prefix to auto-close the issue that your PR addresses

### CODE REVIEW

File: src/constants.commands.ts [hunk 1]
```diff
@@ -171,6 +171,7 @@ export const enum Commands {
 	ShowCommitsView = 'gitlens.showCommitsView',
 	ShowContributorsView = 'gitlens.showContributorsView',
 	ShowDraftsView = 'gitlens.showDraftsView',
+	FocusDraftsView = 'gitlens.views.drafts.focus',
```
Comment (d13) [inline src/constants.commands.ts:174]: this command is no longer used in the walkthrough

File: src/telemetry/walkthroughStateProvider.ts [hunk 2]
```diff
@@ -87,7 +87,16 @@ const walkthroughRequiredMapping: Readonly<Map<WalkthroughContextKeys, Walkthrou
...
-			usage: [`command:${Commands.CreateCloudPatch}:executed`, `command:${Commands.CreatePatch}:executed`],
+			usage: [
+				`patchDetailsView:shown`,
+				`patchDetailsWebview:shown`,
+				`draftsView:shown`,
+				`command:${Commands.ShowDraftsView}:executed`,
+				`command:${Commands.ShowPatchDetailsPage}:executed`,
+				`command:${Commands.CreateCloudPatch}:executed`,
+				`command:${Commands.CreatePatch}:executed`,
+				`command:${Commands.FocusDraftsView}:executed`,
```
Comment (d13) [inline src/telemetry/walkthroughStateProvider.ts:98]: this command is no longer used in the walkthrough

Comment (nzaytsev) [reply src/telemetry/walkthroughStateProvider.ts]: removed

File: src/constants.commands.ts (hunk 1 above)
Comment (nzaytsev) [reply src/constants.commands.ts]: removed

**Pull request merged**